app/views/spree/shared/_ddu_ddp.html.erb

```diff
@@ -1,3 +1,7 @@
+<%
+  @delivered_duty = @order.flow_order ? @order.flow_order.delivered_duty : 'paid'
```

Comment:
would prefer if we used the constaints from the ruby sdk rather than hardcoding `paid` or `unpaid`